auto_tests/reconnect_test.c

```diff
@@ -0,0 +1,104 @@
+/* Auto Tests: Conferences.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "../testing/misc_tools.h"
+#include "../toxcore/friend_connection.h"
+#include "../toxcore/tox.h"
+#include "../toxcore/util.h"
+#include "check_compat.h"
+
+#define NUM_TOXES 2
```

Comment:
This needs a different name, or needs `#undef` at the end: https://travis-ci.org/TokTok/c-toxcore/jobs/416043872#L1251